vmware/docker-volume-vsphere
# Add a test to verify the capacity of the volume created using vmkfstools

Issue #824

Test name: test_VerifyVolumeSize

Test steps:
Create a volume of size 100 mb.
Verify the size of volume using vmkfstools.

Successfully ran the test locally using one esx and two photon vms.
